src/java.base/share/classes/javax/security/auth/Subject.java

```diff
@@ -25,22 +25,18 @@
 
 package javax.security.auth;
 
-import java.util.*;
-import java.io.*;
+import sun.security.util.ResourcesMgr;
```

Comment:
I suggest putting the internal imports below the java.*  or other standard imports. This isn't a rule, but is more consistent with other code.

Reply:
IJ complains about unnecessary imports and asks If I want to let it automatically optimize them. I always answer "yes". I'll fix as you suggest.

Reply:
The default setup wants to put the sun.* imports above the java.*.  The default order is defined in "Import Layout":

```
all other imports
<blank line>
import javax.*
import java.*
<blank line>
import static all other imports
```

I have mine set to:

```
import java.*
import javax.*
<blank line>
all other imports
<blank line>
import static all other imports
```

Reply:
I changed my IJ defaults. Thanks for the suggestion.